cvat-ui/src/components/invitations-page/invitations-list.tsx

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Row, Col } from 'antd/lib/grid';
+import Pagination from 'antd/lib/pagination';
+
+import { CombinedState, InvitationsQuery } from 'reducers';
+import { Invitation } from 'cvat-core/src/organization';
+import { acceptInvitationAsync, rejectInvitationAsync } from 'actions/invitations-actions';
+import { updateHistoryFromQuery } from 'components/resource-sorting-filtering';
+import { useHistory } from 'react-router';
+import notification from 'antd/lib/notification';
+import InvitationItem from './invitation-item';
+
+import dimensions from '../projects-page/dimensions';
+
+interface Props {
+    query: InvitationsQuery;
+}
+
+const PAGE_SIZE = 9;
```

Comment:
Why 9? Is the page size for server request the same? I could not find explicitly defined page size for such requests in client code

Reply:
Answered about pagination in the comment below